snap-extras/sitecustomize.py

```diff
@@ -0,0 +1,8 @@
+import site
```

Comment:
Can this new file or the whole dir to be put inside current `snap` directory? just to not pollute the project's root .